admin/services/pin-generation.service.js

```diff
@@ -98,6 +97,15 @@ pinGenerationService.generateSchoolPassword = (school) => {
   return school
 }
 
+/**
+ * Generate Pupil Pin
+ * @returns {String}
+ */
+pinGenerationService.generatePupilPin = () => {
+  const chars = '23456789'
+  return randomGenerator.getRandom(4, chars)
```

Comment:
Would it not be better to have this argument (the length) in the config file? Same for number 8 in generateSchoolPassword?

Reply:
@pris54 This is a business rule that will apply to all cases. Having it in the config would mean that we would want to differentiate between cases which based on the AC won't really happen.